front-end/src/app/shared/models/contact.model.ts

```diff
@@ -73,42 +73,111 @@ export class Contact extends BaseModel {
   }
 }
 
-export class FecCommitteeLookupData {
+export interface ContactLookupSelectItem<T> extends SelectItem<T> {
+  inContacts: boolean;
+}
+
+export class FecApiLookupData {
+
+}
+export class FecApiCommitteeLookupData extends FecApiLookupData {
   id: string | null = null;
+  is_active: boolean | null = null;
   name: string | null = null;
-  // prettier-ignore
-  static fromJSON(json: any): FecCommitteeLookupData { // eslint-disable-line @typescript-eslint/no-explicit-any
-    return plainToClass(FecCommitteeLookupData, json);
+
+  constructor(data: FecApiCommitteeLookupData) {
+    super();
+    Object.assign(this, data);
   }
-  static toSelectItem(data: FecCommitteeLookupData): SelectItem<string> {
+
+  toSelectItem(): ContactLookupSelectItem<FecApiCommitteeLookupData> {
     return (
-      data && {
-        label: `${data.name} (${data.id})`,
-        value: `${data.id}`,
+      {
+        label: `${this.name} (${this.id})`,
+        value: this,
+        inContacts: false,
+      }
+    );
+  }
+}
+
+export class FecfileCommitteeLookupData extends Contact {
+  constructor(data: FecfileCommitteeLookupData) {
+    super();
+    Object.assign(this, data);
+  }
+
+  toSelectItem(): ContactLookupSelectItem<Contact> {
+    return (
+      {
+        label: `${this.name} (${this.committee_id})`,
+        value: this,
+        inContacts: true,
       }
     );
   }
 }
 
 export class CommitteeLookupResponse {
-  fec_api_committees: FecCommitteeLookupData[] | null = null;
-  fecfile_committees: FecCommitteeLookupData[] | null = null;
+  fec_api_committees: FecApiCommitteeLookupData[] | null = null;
```

Comment:
I'd recommend changing this to reflect what could come from the server (either an array or undefined).  The server never returns a fec_api_committees of value `null`.
`fec_api_committees: FecApiCommitteeLookupData[] | undefined;`